app/forms/provider_interface/offer_wizard.rb

```diff
@@ -123,5 +122,21 @@ def update_path_history(attrs)
       @wizard_path_history.update
       @path_history = @wizard_path_history.path_history
     end
+
+    def setup_further_conditions(attrs)
+      self.further_conditions = 4.times.to_a.to_h do |id|
+        [id, attrs.dig('further_conditions', id.to_s).presence || { 'text' => '' }]
+      end
+    end
```

Comment:
This is a bit messy. The intention is to parameterize how many boxes we want, and then to populate the first boxes with values that we get from params

Reply:
I was wondering if we can move the determination of how many conditions are rendered to the views, and as far as the wizard is concerned it just validates what it gets. This makes the logic here simpler. And its a smaller step when we integrate the add another feature which will be determining that as well?

Reply:
I'm not sure we can keep it just to the view, since the view needs a number of already created OfferConditions models in order to render the fields_for

Reply:
I think the problem here is that it's trying to do more than it needs to at the moment. Simpler version to come